store/transient/store.go

```diff
@@ -40,7 +40,11 @@ func (ts *Store) GetPruning() pruningtypes.PruningOptions {
 
 // Implements CommitStore
 func (ts *Store) LastCommitID() (id types.CommitID) {
-	return
+	return id
+}
+
+func (ts *Store) WorkingHash() (hash []byte) {
```

Comment:
Why using named return? It makes it harder to read here.

Reply:
https://github.com/cosmos/cosmos-sdk/pull/15712#discussion_r1159943554

Reply:
Yes, but it was about the naked return. I am talking about the named return value:
IMHO `func Foo() (v int) { return v }` is less readable than `func Foo() int { return 0 }`

Reply:
Why is that harder to read? IMO there's nothing wrong with named return values, especially in concise functions/methods. It's _naked_ returns that are a big no no IMO.

Reply:
In my thought process, that's one extra step (quickly thinking what was `v` and its zero value). While without the named return, it is immediately clear.